webpack.mix.js

```diff
@@ -24,4 +24,8 @@ mix.styles([
     'resources/css/Post/style.css',
 ], 'public/css/Post/style.css');
 
+mix.styles([
```

Comment:
sửa lại tên folder nha e, không viết in hoa

Reply:
Dạ vâng ạ